core/src/main/java/io/jenkins/servlet/http/HttpSessionEventWrapper.java

```diff
@@ -0,0 +1,16 @@
+package io.jenkins.servlet.http;
+
+import java.util.Objects;
+import javax.servlet.http.HttpSessionEvent;
+
+public class HttpSessionEventWrapper {
```

Comment:
> How is this used?

I think the main usage of this is in `sse-gateway` plugin. At least I seem to recall PCT failing for that until I implemented a compatibility layer here.

> What about subclasses like https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpSessionBindingEvent.html ?

What about them?

Reply:
I forgot to delete this one; I found since then that while they're related types, they're used in entirely different contexts, so it's unlike the (Http)ServletRequest/Response situation necessitating `instanceof`.